app/src/main/res/values/strings.xml

```diff
@@ -657,6 +657,8 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_bicycle_parking_type_building">Building</string>
     <string name="quest_bicycle_parking_type_handlebarholder">Handlebar holder</string>
 
+    <string name="quest_bicycle_rental_capacity_title">How many bikes can be rented or returned here?</string>
```

Comment:
@ Native speakers (@peternewman @smichel17) wording suggestions? I'm not quite convinced with my suggestion yet.

Reply:
Yeah that wording would worry me that people would either try and count the bikes currently docked or the free slots.

How about (modifying the existing bike quest):
```suggestion
    <string name="quest_bicycle_rental_capacity_title">How many rental bikes can be parked here?</string>
```
```suggestion
    <string name="quest_bicycle_rental_capacity_title">How many rental bike spaces are there here?</string>
```
```suggestion
    <string name="quest_bicycle_rental_capacity_title">How many rental bike docks are there here?</string>
```

Reply:
> How many rental bike spaces are there here?

"there here"?

> How many rental bike docks are there here?

I am not sure if "dock" would be the most common word for rental stations. 

I find the original text or the 2nd suggestion (without "there here") best currently. Though maybe something with "at maximum capacity" would work too?

Reply:
> (in reply to @peternewman)
> 
> > How many rental bike spaces are there here?
> 
> "there here"?

"there here" is acceptable to me, but it would probably also work without the *here* (as long as the name/type of the docking station is presented).

> 
> > How many rental bike docks are there here?
> 
> I am not sure if "dock" would be the most common word for rental stations.
> 
> I find the original text or the 2nd suggestion (without "there here") best currently. Though maybe something with "at maximum capacity" would work too?

Reply:
Changed in [`4515039`](https://github.com/streetcomplete/StreetComplete/pull/4008/commits/4515039b68a65d84cc6c7b8893aedc1bf70eab32).

> as long as the name/type of the docking station is presented

It is, just like in every other quest.